
Test case ID:|Module:View Self Details|Description:"To verfy validation message appears ""Login Id is not available in request"" when LOGIN_ID   not provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"Login Id or External code is not available in request","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation messsage appears ""Password is not available in request"" when password is not provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"Password is not available in request","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""User does not exists in the system"" when wrong LOGIN_ID provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"User does not exists in the system","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""User ID or Password is incorrec"" when wrong PASSWORD provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"User ID or Password is incorrect","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""Action type does not exists in the request"" when TYPE is not provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"Action type does not exists in the request","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""invalid.login.id"" when provide invalid login id"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"viewSelfDetails.login.invalid","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""login.id.required"" when do not provide dataLoginID"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"viewSelfDetails.login.required","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verfy validation message appears ""Login Id is not available in request"" when LOGIN_ID   not provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"Login Id or External code is not available in request","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation messsage appears ""Password is not available in request"" when password is not provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"Password is not available in request","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""User does not exists in the system"" when wrong LOGIN_ID provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"User does not exists in the system","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""User ID or Password is incorrec"" when wrong PASSWORD provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"User ID or Password is incorrect","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""Action type does not exists in the request"" when TYPE is not provided"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"Action type does not exists in the request","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""invalid.login.id"" when provide invalid login id"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"viewSelfDetails.login.invalid","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
Test case ID:|Module:View Self Details|Description:"To verify validation message appears ""login.id.required"" when do not provide dataLoginID"|Request:null|Response:{"statusCode":400,"status":false,"successMsg":null,"formError":"viewSelfDetails.login.required","globalError":null,"parameters":null,"dataObject":null,"fieldError":null,"messageCode":null}|Status:FAIL
